Flow.Launcher.Plugin.PythonTemplate/commands.py

```diff
@@ -72,17 +101,92 @@ def gen_plugin_info():
         "Language": PLUGIN_PROGRAM_LANG,
         "Website": PLUGIN_URL,
         "IcoPath": ICON_PATH,
-        "ExecuteFileName": PLUGIN_EXECUTE_FILENAME
+        "ExecuteFileName": PLUGIN_EXECUTE_FILENAME,
     }
 
-    json_path = os.path.join(basedir, 'plugin.json')
-    with open(json_path, 'w') as f:
-        json.dump(plugin_infos, f, indent=' '*4)
+    with open(basedir / "plugin.json", "w") as f:
+        json.dump(plugin_infos, f, indent=4)
+
+    click.echo("Done.")
+
+
+@plugin.command()
+def build():
+    "Pack plugin to a zip file."
+
+    # zip plugin
+    build_path = basedir / "build"
+    build_path.mkdir(exist_ok=True)
+    zip_path = build_path / f"{__package_name__.title()}-{__version__}.zip"
+    zip_path.unlink(missing_ok=True)
+
+    ignore_list = [
+        # folder
+        ".git/*",
+        ".vscode/*",
+        ".history/*",
+        "*/__pycache__/*",
+        "build/*",
+        # file
+        ".gitignore",
+        ".gitattributes",
+    ]
+    os.system(f"zip -r {zip_path} . -x {' '.join(ignore_list)}")
+
+    # hook lib folder path to python system environment variable path
+    env_snippet = dedent(
+        """\
+    import os
+    import sys
+
+    basedir = os.path.dirname(os.path.abspath(__file__))
+    sys.path.append(os.path.join(basedir, "lib"))
+    """
+    )
+
+    entry_src = basedir / "main.py"
+    entry_src_temp = build_path / "main.py"
+    with open(entry_src, "r") as f_r:
+        with open(entry_src_temp, "w", encoding="utf-8") as f_w:
+            f_w.write(env_snippet + f_r.read())
+
+    os.system(f"zip -j {zip_path} {entry_src_temp}")
+    entry_src_temp.unlink()
+
+    click.echo("Done.")
+
+
+@click.group()
+def clean():
+    """Clean commands."""
+    ...
+
+
+@clean.command()
+def clean_build():
+    """Remove build artifacts"""
+
+    os.system("rm -fr build/")
+    click.echo("Done.")
+
+
+@clean.command()
+def clean_pyc():
+    "Remove Python file artifacts"
 
-    click.echo('Done.')
+    os.system(f"find {basedir} -name '*.pyc' -exec rm -f {{}} +")
```

Comment:
Actually, the `find` command also doesn't exist for window.
I run it in ps env with [choco](https://chocolatey.org/) to install these commands.

But those (ex. zip, find) commands will run at CI Linux env, so don't worry about it.